var/spack/repos/builtin/packages/spla/package.py

```diff
@@ -15,18 +15,28 @@ class Spla(CMakePackage):
     url      = "https://github.com/eth-cscs/spla/archive/v1.0.0.tar.gz"
     git = 'https://github.com/eth-cscs/spla.git'
 
+    version('1.1.0', sha256='b0c4ebe4988abc2b3434e6c50e7eb0612f3f401bc1aa79ad58a6a92dc87fa65b')
     version('1.0.0', sha256='a0eb269b84d7525b223dc650de12170bba30fbb3ae4f93eb2b5cbdce335e4da1')
+    version('develop', branch='develop')
     version('master', branch='master')
 
     variant('openmp', default=True, description="Build with OpenMP support")
     variant('static', default=False, description="Build as static library")
-    variant('cuda', default=False, description="CUDA")
+    variant('cuda', default=False, description="CUDA backend")
+    variant('rocm', default=False, description="ROCm backend")
+
+    conflicts("spla+cuda", when="+rocm", msg="+cuda and +rocm are mutually exclusive")
```

Comment:
```suggestion
    conflicts("+cuda", when="+rocm", msg="+cuda and +rocm are mutually exclusive")
```

Reply:
Thanks for the correction, I've included the change.